pkgs/applications/networking/instant-messengers/webcord/webcord-vencord/default.nix

```diff
@@ -2,9 +2,12 @@
 , substituteAll
 , lib
 , vencord-web-extension
+, electron_24
```

Comment:
this probably can be replaced with `electron ? electron_24` in order to get rid of the redundant let in

Reply:
I don't really understand

doing  `electron ? electron_24,` results in variable `electron_24` not found.
and either way it wouldn't care about the `? electron_24` part because callPackage will assign electron to electron_26 (cuz the packacge already exists)

I think we should keep it as `electron_25 = electron_24` for now. The only problem with it that I can think of is that when webcord changes electron versions, that line will no longer do anything. But no idea how to solve it and this probably shouldn't be dealt with in this PR..

Reply:
correction, pkgs.electron_24*

Reply:
Ugghh, Got this error using `pkgs.electron_24`
```console
       error: undefined variable 'pkgs'

       at /home/ludovico/Code/nixpkgs/pkgs/applications/networking/instant-messengers/webcord/webcord-vencord/default.nix:5:14:

            4| , vencord-web-extension
            5| , electron ? pkgs.electron_24
             |              ^
            6| }:
```

I think it's just better to use the old 
```nix
electron_25 = electron_24
```

Reply:
Why don't we do something like what java is doing with jre? Let the package accept `electron` and set `electron = electron_xx;` in callPackage in `top-level/all-packages.nix`.

For webcord-vencord, we could do `callPackage ... { webcord = webcord.override { electron = electron_xx; }; }`

Reply:
Good idea. That will be better than just doing `electron_25 = electron_24` if Webcord maintainers are alright with it. But I will approve the changes anyway(after removing the let in) since the point of this PR is to just override the electron version to something else like in previous commits.